_shows/75_76/the_leader.md

```diff
@@ -0,0 +1,28 @@
+---
+title: The Leader
+playwright: Eugène Ionesco
+season: In House
+period: Spring
+season_sort: 200
+venue: New Theatre
+
+cast:
+- role: The Announcer
+  name: Richard Truman
+- role: The Young Lover
+  name: Neil Runcieman
+- role: The Girlfriend
+  name: Terri Norris
+- role: The Aclminer
+  name: Matthew Bannister
+- role: The Girl Aclminer
+  name: Melanie Comley
+- role: The Leader
+  name: Simon Holland
+
+crew:
+- role: Director
+  name: Heather Peace
+---
+
+In 'The Leader' Ionesco demonstrates, in his customary indirect style, the absurdity of human behaviour and social anganisation. 
```

Comment:
Anganisation isn’t a word. I can only suggest antagonisation. Potentially adding a comment if we really have no idea?